tests/Database/SampleFiles/SampleClass.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace October\Rain\Tests\Database\SampleFiles;
+
+use PHPUnit\Framework\TestCase;
```

Comment:
@iulianwebdev Might be safer to use some random class in the October CMS Rain library (perhaps `October\Rain\Supports\Arr`) as opposed to TestCase.

Reply:
Followed your advice and used the Array utility library.